configs/test_indexes.json

```diff
@@ -0,0 +1,851 @@
+{
```

Comment:
Lets use the `golang-standards/project-layout`

https://github.com/golang-standards/project-layout#test